### ISSUE TITLE
ontology prefix convention

### ISSUE THREAD

**Issue by loicjaouen** [dhlab-basel/Knora#632]
we should decide and document the ontology prefix to be used for:

- resource classes:
  as in [anything-onto.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/ontologies/anything-onto.ttl):

  ```
  http://www.knora.org/ontology/anything#Thing
  ```

- property declaration:
  as in [anything-onto.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/ontologies/anything-onto.ttl):

  ```
  http://www.knora.org/ontology/anything#hasBlueThingValue
  ```

- resource instances
  as in [anything-data.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/all_data/anything-data.ttl):

  ```
  <http://data.knora.org/0C-0L1kORryKzJAJxxRyRQ> a anything:Thing ;
  ```

- values
  as in [anything-data.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/all_data/anything-data.ttl):

  ```
  <http://data.knora.org/a-thing-with-text-values/values/1> a knora-base:TextValue ;
  ```

- lists
  as in [anything-data.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/all_data/anything-data.ttl):

  ```
  <http://data.knora.org/anything/treeList> a knora-base:ListNode ;
  ```

- user
  as in [admin-data.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/all_data/admin-data.ttl):

  ```
  <http://data.knora.org/users/BhkfBc3hTeS_IDo-JgXRbQ> a knora-base:User ;
  ```

- group
  as in [drawings-gods_admin-data.ttl](https://github.com/dhlab-basel/Knora/blob/develop/webapi/_test_data/other.v1.DrawingsGodsV1Spec/drawings-gods_admin-data.ttl):

  ```
  <http://data.knora.org/groups/drawings-gods-meta-annotators> rdf:type knora-base:UserGroup ;
  ```

- permissions  
  as in [permissions-data.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/all_data/permissions-data.ttl):

  ```
  <http://data.knora.org/permissions/005-a1>
  ```

- knora-projets
  as in `admin-data.ttl`:

  ```
  <http://data.knora.org/projects/b83b99ca01> a knora-base:knoraProject ;
  ```  

- projectOntologyGraph
  as in [admin-data.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/all_data/admin-data.ttl):

  ```
  knora-base:projectOntologyGraph "http://www.knora.org/ontology/anything"^^xsd:string ;
  ```  

- projectDataGraph
  as in [admin-data.ttl](https://github.com/dhlab-basel/Knora/blob/8c1c0405e1616aa0b5a59cb4f79d81b787c2c6d4/webapi/_test_data/all_data/admin-data.ttl):

  ```
  knora-base:projectDataGraph "http://www.knora.org/data/anything"^^xsd:string ;
  ```

**Comment by benjamingeer** [comment]
I'll do this after #631 is merged.

**Comment by loicjaouen** [comment]
An idea is to set `knora.org` for knora specific notions, for example:
- we store data in knora, named graphs can remain as `knora.org`
- resources should be able exist even if we replace knora one day, so a resource instance should be prefixed with `rdfh.ch`

@benjamingeer pointed out that since our last conversation on the subject, all new resources created in knora have the `rdfh.ch` prefix:
`http://rdfh.ch/projectid/-CkgZsCXTxir0RUqfrP_qg`

**Comment by mrivoal** [comment]
As I am moving forward to a new ontology, I would like an update on this. 
General guidelines, as we discussed, would be awesome, but right now I would like to know what do we do with **lists**? 

Until now, we (in Lausanne) have moved lists from  `<http://data.knora.org/anything/treeList>` to `<http://rdfh.ch/lists/treeList>`. But given our last discussion, I am not sure anymore that this is the right thing to do. 

So, what do you think?
* `http://rdfh.ch` instead of `http://data.knora.org`?
* adding a `/list` segment to the IRI might (IMHO) be a good thing because it makes it easier to discriminate lists from other data within the data named graph.
* having the ID of the project included in the IRIs syntax also makes sense (as in Anything project) as long as lists are not shared between projects.

So, should we have something like `<http://rdfh.ch/anything/lists/treeList>` ?

**Comment by benjamingeer** [comment]
@mrivoal Yes, I think you're right. I'll be writing documentation about IRIs tomorrow.

**Comment by mrivoal** [comment]
Wonderful, thanks! :)
